flutter/test/sentry_flutter_util.dart

```diff
@@ -6,36 +6,35 @@ import 'package:sentry/sentry.dart';
 import 'package:sentry_flutter/sentry_flutter.dart';
 import 'package:sentry_flutter/src/file_system_transport.dart';
 import 'package:sentry_flutter/src/version.dart';
+
 import 'mocks.dart';
 
-FutureOr<void> configurationTester(
-  SentryOptions options, {
-  bool isWeb = false,
-}) async {
-  options.dsn = fakeDsn;
+FutureOr<void> Function(SentryOptions) getConfigurationTester(bool onIOS) =>
+    (SentryOptions options) async {
+      options.dsn = fakeDsn;
 
-  expect(kDebugMode, options.debug);
-  expect('debug', options.environment);
+      expect(kDebugMode, options.debug);
+      expect('debug', options.environment);
 
-  expect(true, options.transport is FileSystemTransport);
+      expect(true, options.transport is FileSystemTransport);
 
-  expect(
-      options.integrations
-          .where((element) => element == flutterErrorIntegration),
-      isNotEmpty);
+      expect(
+          options.integrations
+              .where((element) => element == flutterErrorIntegration),
+          isNotEmpty);
 
-  expect(
-      options.integrations
-          .where((element) => element == isolateErrorIntegration),
-      isNotEmpty);
+      expect(
+          options.integrations
+              .where((element) => element == isolateErrorIntegration),
+          isNotEmpty);
 
-  expect(4, options.integrations.length);
+      expect(onIOS ? 5 : 4, options.integrations.length);
```

Comment:
I think this can get complicated to test if its iOS, if its Android, if its Web, how can we improve this, maybe doing an assertion by integration name?

```
if (iOS)
  expect(options.integrations.contains(name), true)
```

etc...

Reply:
✅ I let the integration number test and added new to test "by integration name"